cmd/build.go

```diff
@@ -60,7 +60,8 @@ func newBuildCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	}
 
 	buildCmd.PersistentFlags().StringVarP(&config.tag, "tag", "t", "", "Docker image name and optionally a tag in the 'name:tag' format")
-	buildCmd.PersistentFlags().StringVar(&config.dockerBuildOptions, "docker-options", "", "Specify the docker build options to use.  Value must be in \"\".")
+	buildCmd.PersistentFlags().BoolVar(&rootConfig.Buildah, "buildah", false, "Build project using buildah primitives instead of docker.")
```

Comment:
@chilanti and I tossed around the idea of deprecating `--docker-options` and using a new flag `--opts` or `--options` for both docker and buildah.

However, those flags are not very clear. I like the explicit nature of `--docker-options` and `--buildah-options` so that's what I've gone with for now.